web-gateway/app/views/editItem.scala.html

```diff
@@ -1,14 +1,30 @@
 @import helper._
```

Comment:
They are diverging and I think they will continue to diverge. Let's see what can we reuse in a few days.